src/net.h

```diff
@@ -802,9 +802,14 @@ class V1TransportDeserializer final : public TransportDeserializer
         hdrbuf.SetVersion(nVersionIn);
         vRecv.SetVersion(nVersionIn);
     }
-    int Read(const char *pch, unsigned int nBytes) override {
-        int ret = in_data ? readData(pch, nBytes) : readHeader(pch, nBytes);
-        if (ret < 0) Reset();
+    int Read(Span<const char>& msg_bytes) override
```

Comment:
Might want to add a comment here that this function reads bytes from the span and advances the pointer. At least I first missed where the `msg_bytes` was advanced in the `CNode::ReceiveMsgBytes` loop.

Reply:
thx, done